core/core.go

```diff
@@ -80,6 +83,9 @@ type TextileNode struct {
 
 	// Signal we left the room
 	LeftRoomCh chan struct{}
+
+	// The local password used to authenticate http gateway requests (username is TextileNode)
+	Password string
```

Comment:
how about GatewayPassword? We'll have other passwords?

Reply:
Yup, good call